docs/user-guide/configuration.md

```diff
@@ -3,31 +3,32 @@ id: configuration
 title: Configuration
 ---
 
-Search `.htmlhintrc` file in current directory and all parent directories:
+By default, htmlhint looks for a `.htmlhintrc` file in the current directory
+and all parent directories, and applies its rules when parsing a file:
```

Comment:
Please also remove the `,` comma in front of the `and`

Reply:
I can do that, but why should I? It is used here to provide a semantic pause between two relatively long clauses to facilitate reading. That's legitimate use of the punctuation, as described [here](https://en.wikipedia.org/wiki/Comma#Separation_of_clauses).

Reply:
Wow, I like working with collaborators who can provide explanations